frontend/simple/views/i18n.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <span><slot></slot></span>
```

Comment:
At some point in the future, maybe Vue's render function can be used instead of this template mechanism which seems to require a tag (in this case `<span>`) which we don't really need.